app/views/studies/_expression_file_info_fields.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="row expression-file-info-fields">
+  <div class="col-sm-2">
+    <%= f.label :is_raw_counts, 'Raw Count Matrix?' %><br />
```

Comment:
For consistency:

```suggestion
    <%= f.label :is_raw_counts, 'Raw Counts Matrix?' %><br />
```

Reply:
If it indeed should be labeled "raw count", I'd suggest we rename this throughout our code.  Tedious, but best to be consistent very early and before we release.

Reply:
7ff60c2